t/35-script_clone_job_suse.t

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018-2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use Test::Warnings ':report_warnings';
+
+use FindBin;
+use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '6';
+use Mojo::Base -signatures;
+use Test::MockModule;
+use OpenQA::Script::CloneJobSUSE;
+use Mojo::URL;
+
+# define fake client
+{
+    package Test::FakeLWPUserAgentMirrorResult;
+    use Mojo::Base -base, -signatures;
+    has is_success => 1;
+}
+
+subtest 'maintenance update detect' => sub {
+    my $job_id = 1;
+    my %job = (
+        id => $job_id,
+        BASE_TEST_TEMPLATE => 'http://foo/@INCIDENTNR@/SLE-Module-Basesystem/',
+        BASE_TEST_ISSUES => '24266,24314,24379,24392,24404',
+
+    );
+    my %response = ('is_success' => 1,);
+    my $clone_mock = Test::MockModule->new('LWP::UserAgent');
+    $clone_mock->mock(get => sub { return Test::FakeLWPUserAgentMirrorResult->new(\%response) });
```

Comment:
```suggestion
    $clone_mock->redefine(get => sub { return Test::FakeLWPUserAgentMirrorResult->new(\%response) });
```

Reply:
I have modified the test. Please let me know if this is okay? Thank you.